dotcom-rendering/src/web/lib/dailyArticleCount.ts

```diff
@@ -12,11 +12,12 @@ interface DailyArticleCountLocalStorage {
 
 export const DailyArticleCountKey = 'gu.history.dailyArticleCount';
 
-export const getDailyArticleCount = (): DailyArticleCount => {
+// Returns undefined if no daily article count in local storage
+export const getDailyArticleCount = (): DailyArticleCount | undefined => {
```

Comment:
I changed the return type here because `undefined` is a more deliberate signal that there is _no_ article count. This is consistent with the weekly article count.